.github/workflows/build-ironic-images.yaml

```
@@ -0,0 +1,35 @@
+---
+name: build-ironic-images
+
+on:
+  workflow_dispatch:
+  pull_request:
+    paths:
+      - 'ironic-images/**'
+  push:
+    branches:
+      - main
+    paths:
+      - 'ironic-images/**'
+
+jobs:
+  build-ironic-images:
+    runs-on: ubuntu-latest
```

```suggestion
    runs-on: ubuntu-latest
    defaults:
      run:
        working-directory: ironic-images/ipa-debian-bookworm
```

would remove the need for the `cd` in all the commands.

---

Thanks 👍